source/options.html

```diff
@@ -44,8 +44,8 @@
 	</details>
 
 	<details id="action">
-		<summary><strong>🔗 Refined GitHub link</strong></summary>
-		<p>This page will open when clicking the toolbar icon</p>
+		<summary><strong>🔗 Button link</strong></summary>
+		<p>You can pick what page opens when you click the Refined GitHub icon in the browser toolbar</p>
```

Comment:
I tried to improve this section to avoid repetition and clarify what icon and what toolbar we're talking about.